lib/mix/tasks/phx.gen.notifier.ex

```diff
@@ -164,4 +177,44 @@ defmodule Mix.Tasks.Phx.Gen.Notifier do
     |> files_to_be_generated()
     |> Mix.Phoenix.prompt_for_conflicts()
   end
+
+  @doc """
+  Print mailer instructions if mailer is not defined.
+
+  This is useful for applications there were created without the
+  mailer.
+  """
+  @spec maybe_print_mailer_installation_instructions(%Context{}) :: %Context{}
+  def maybe_print_mailer_installation_instructions(%Context{} = context) do
+    mailer_module = Module.concat([context.base_module, "Mailer"])
+
+    unless Code.ensure_loaded?(mailer_module) do
+      Mix.shell().info("""
+      Unable to find the "#{inspect(mailer_module)}" module defined.
+
+      A mailer module like the following is expected to be defined
+      in your application in order to send emails.
+
+          defmodule #{inspect(mailer_module)} do
+            use Swoosh.Mailer, otp_app: #{inspect(context.context_app)}
+          end
+
+      It is also necessary to add "swoosh" as a dependency in your
+      "mix.exs" file:
+
+          def deps do
+            [{:swoosh, "~> 1.4"}]
+          end
+
+      An adapter needs to be set in your configuration:
+
+          use Mix.Config
```

Comment:
sorry if this was brought up already, can we replace Mix.Config with Elixir Config going forward? Or Phoenix itself (different than newly generated Phoenix app!) supports older version?

```suggestion
          import Config
```

Reply:
Good call, that's the default in new apps already.